components/vf-form__select/index.scss

```diff
@@ -8,5 +8,5 @@
 
 // component specific styles
 
-@import 'vf-form__core/vf-form__core.scss';
+@import 'vf-form/vf-form.scss';
```

Comment:
Just adding a non-mental note that we'll want to highlight this change in the /updates notes.

Reply:
It might be good to make a longer post for this release too?

Reply:
Yeah. I'm sure screenshots would be welcome. We might also reference the process/reasons/consultation by which these changes came about.

Figure you'll want to write that up? Happy to assist where helpful.